### ISSUE TITLE
[a11y]: Toggle The SVG element has no accessible name

### ISSUE THREAD

**Issue by AlvinChacko** [carbon-design-system/carbon#17223]
### Package

@carbon/react

### Browser

Firefox

### Operating System

MacOS

### Package version

v1.62.1

### React version

v16.12.0

### Automated testing tool and ruleset

IBM Equal Access Accessibility Checker

### Description

The small toggle had an SVG with no label. Either set it as a aria-hidden or add a label. Here is where I saw that:

https://github.com/carbon-design-system/carbon/blob/87d4643e49b18dacb1ee02dc0af92e88021ad6f6/packages/react/src/components/Toggle/Toggle.tsx#L206

### WCAG 2.1 Violation

IBM 1.1.1 Non-text Content

### Reproduction/example

https://stackblitz.com/edit/github-cbzsy3?file=src%2FApp.jsx

### Steps to reproduce

Create a toggle element with size sm and scan the toggle using accessibility checker.

### Suggested Severity

None

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by alisonjoseph** [comment]
This fails with latest `06Aug2024` ruleset

**Comment by 2nikhiltom** [comment]
Hey! It seems you'll need to open the component in a separate iframe and run an accessibility scan. This ensures only the component is tested. I didn't encounter the issue during my scan. The violations you're seeing might be caused by testing it directly in Storybook.

https://github.com/user-attachments/assets/7d5e818c-c9f8-469a-8b9c-d06eee1782c4

**Comment by AlvinChacko** [comment]
Okay I believe this was fixed recently because when I tested it in an separate iframe it was still there in Aug.
